src/modules/utils.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+/**
+ * A function to generate a pause in promise chaining
+ * @param  {number} timer How much ws to wait
+ * @return {Promise}      A Promise that will resolve when timer is expired
+ */
+module.exports.timeoutPromise = (timer) => {
+  return new Promise((resolve) => {
+    setTimeout(resolve, timer);
```

Comment:
I want to use a specific amount of time, like 10 seconds. Can I achieve that using nextTick without doing a recursive loop with increment counter?